recipes/r-rcppplanc/conda_build_config.yaml

```diff
@@ -0,0 +1,14 @@
+# Please consult conda-forge/core before doing this
+c_stdlib_version:
+  - "11"           # [osx and x86_64]
+  - "2.17"         # [linux and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "11.3"                # [osx and x86_64]
+blas_impl:
+  - openblas     # [not osx]
+  - mkl          # [(x86 or x86_64) and not osx]
+  - accelerate   # [osx]
```

Comment:
Please use the generic `blas_impl`, as there is no MKL specific code in `rcppplanc`.

Reply:
Could have sworn there was something going on in the cblas_sgemm override that was modified for mkl, bear with.

Reply:
Okay, I'm not going crazy, there _is_ something in upstream. I stripped it out because the nmf side of things doesn't use it. Will fix.